### ISSUE TITLE
Opni agent helm values Secret ref comment invalid

### ISSUE THREAD

**Issue by ron1** [rancher/opni#1675]
The following opni-agent helm values file comment is no longer valid and therefore confusing.

https://github.com/rancher/opni/blob/2a4061416dc7b03792385fb9c28e009ab9ea62ea/packages/opni-agent/opni-agent/charts/values.yaml#L71-72

Either remove the comment or restore the former conditional Secret generation behavior.

**Comment by kralicky** [comment]
Good idea. We can add an option to the chart to skip creating the agent config secret.

Side note, only the bootstrap token is a secret (and they can be revoked by the server at any time). The pin is a public key.

**Comment by ron1** [comment]
Thanks for the clarification. I didn't realize the pin was a public key.
